sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizeEntitiesResultCollection.cs

```diff
@@ -7,25 +7,28 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
+    /// Collection of <see cref="RecognizeEntitiesResult"/> objects corresponding
+    /// to a batch of input documents, and annotated with information about the
+    /// batch operation.
     /// </summary>
     public class RecognizeEntitiesResultCollection : ReadOnlyCollection<RecognizeEntitiesResult>
     {
-        /// <summary>
-        /// </summary>
-        /// <param name="list"></param>
-        /// <param name="statistics"></param>
-        /// <param name="modelVersion"></param>
         internal RecognizeEntitiesResultCollection(IList<RecognizeEntitiesResult> list, TextDocumentBatchStatistics statistics, string modelVersion) : base(list)
         {
             Statistics = statistics;
             ModelVersion = modelVersion;
         }
 
         /// <summary>
+        /// Gets statistics about the input document batch and how it was processed
+        /// by the service.  This property will have a value when IncludeStatistics
+        /// is set to true in the client call.
         /// </summary>
         public TextDocumentBatchStatistics Statistics { get; }
 
         /// <summary>
+        /// Gets the version of the text analytics model used to identify
```

Comment:
I wonder if it would make sense to not mention "entities" in the comment here since both statistics and model versions are common for every result type. It would make it much more plausible for someone to copy/paste this class and not review the comment. WDYT?

Reply:
Cool, yes, I can rework that.  Thanks!